### DISCUSSION TITLE
How to change version on Laragon from 8+ to 7.4 stable version

### DISCUSSION THREAD

**Discussion by Hilal-Zikra** [leokhoa/laragon#634] [Q&A]
Need to change my laragon PHP version from 8+ to 7.4 stable version

**Comment by Pen-y-Fan** [comment]
See my blog [How to update the PHP version in Laragon](https://pen-y-fan.github.io/2023/01/15/how-to-update-the-php-version-in-laragon/) for details.

PHP 7.4 is now in the [windows PHP download archive](https://windows.php.net/downloads/releases/archives/) section. Search for **php-7.4.32** and download the version required e.g.:

- 64-bit NTS version **php-7.4.32-nts-Win32-vc15-x64.zip**  or
- 64-bit TS version **php-7.4.32-Win32-vc15-x64.zip**

**Reply by tertek** [reply]
Thank you for this!

**Reply by asif-ca** [reply]
It was useful thanks
